signac/contrib/schema.py

```diff
@@ -1,6 +1,7 @@
-# Copyright (c) 2017 The Regents of the University of Michigan
+# Copyright (c) 2017 The Regents of the University of Michigan.
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
+"""Project Schema."""
```

Comment:
I like putting a newline after the module docstring. I think that's a matter of preference.